test/integration/kueuectl/list_test.go

```diff
@@ -229,9 +229,9 @@ very-long-workload-name                         lq1
 wl1                                             lq1                                         PENDING                       %s
 wl2                                             very-long-local-queue-name                  PENDING                       %s
 `,
+					duration.HumanDuration(executeTime.Sub(wl3.CreationTimestamp.Time)),
```

Comment:
I don't think that this should be resolved the root cause since the metav1 sore the only second level time, right?
So, could you add a sleep for every workload creation?

```shell
 lq1 = testing.MakeLocalQueue("lq1", ns.Name).ClusterQueue("cq1").Obj() 
 gomega.Expect(k8sClient.Create(ctx, lq1)).To(gomega.Succeed()) 
 time.Sleep(time.Second)
...
```

https://github.com/kubernetes-sigs/kueue/blob/ccfadf0a87dbb2043afb0dc6db4c862b8fe77cec/test/integration/kueuectl/list_test.go#L60-L67

Reply:
Ah got it. Thanks.